config/routes.rb

```diff
@@ -15,8 +15,8 @@
   get 'auth/:provider/callback', to: 'sessions#create'
   resources :tweets, only: [:new, :create]
 
-	namespace :stores, as: :store, path: ":store" do
-		resources :items, only: [:index, :show, :new, :create, :update, :edit]
+  namespace :stores, as: :store, path: ":store" do
+    resources :items, only: [:index, :show, :new, :create, :update, :edit]
```

Comment:
I think `/store-slug` can be the items `index` instead of namespacing it.

`show` can probably just be `/items/:id`. Don't think it needs to be namespaced.